applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -377,6 +522,23 @@ static void stream_stopped_cb(struct bt_audio_stream *stream)
 		ret = ctrl_events_le_audio_event_send(LE_AUDIO_EVT_NOT_STREAMING);
 		ERR_CHK(ret);
 	}
+
+	if (stream->user_data != NULL) {
```

Comment:
```suggestion
	if (stream->user_data == NULL) {return;}
```

Reply:
Remove indentations where we can for readability.

Reply:
This is no longer relevant as the code has already changed.